_locales/en/messages.json

```diff
@@ -67,6 +67,10 @@
     "message": "Always on top",
     "description": "The label for the checkbox in settings tab that sets the window always on top."
   },
+  "accessibilityModeSetting": {
+    "message": "Accessibility Mode",
+    "description": "The label for the checkbox in settings tab that activates accessibility mode, this mode allows for screenreaders to function correctly but disables syntax highlighting"
```

Comment:
how is it that dev tools keeps the syntax highlighting?

Reply:
it doesn't have proper screen reader support